### PR TITLE
tweak markdown style in docs.md

### PR DESCRIPTION
**Pull request by liadbiz**
detail:

+ change line-height of all `p` element and li element to `2rem` , it looks nicer
+ style all `blockquote`
+ add border-radius to all `code` block
+ remove `posiition: fixed` of footer component

Any suggestions are welcome to make the the the raw html markup more beautiful are welcome :smile:

### CODE REVIEW

File: pages/docs.js [hunk 1]
```diff
@@ -31,7 +31,29 @@ const MarkdownWrapper = glamorous.div({
   '& a': {
     textDecoration: 'none',
     color: colors.primary
-  }
+  },
+  '& p': {
+    'line-height': '2rem',
```
Comment (paulmolluzzo) [inline pages/docs.js]: Multiword CSS properties should be camelcased. You can make this `lineHeight`

File: pages/docs.js [hunk 2]
```diff
@@ -31,7 +31,29 @@ const MarkdownWrapper = glamorous.div({
...
+    '& code': {
+      padding: '.3rem',
+      'border-radius': '.3rem'
```
Comment (paulmolluzzo) [inline pages/docs.js]: `borderRadius`

File: pages/docs.js [hunk 3]
```diff
@@ -31,7 +31,29 @@ const MarkdownWrapper = glamorous.div({
...
+  '& blockquote': {
+    margin: '1rem 0',
+    padding: '0 1rem',
+    color: '#666',
+    'border-left': '6px solid #ddd'
+  },
+
+  '& li': {
+    'line-height': '2rem',
+    "& code": {
```
Comment (paulmolluzzo) [inline pages/docs.js]: Please fix mixed quote styles.

File: pages/docs.js [hunk 4]
```diff
@@ -31,7 +31,29 @@ const MarkdownWrapper = glamorous.div({
...
+       padding: '.3rem',
+      'border-radius': '.3rem'
+    }
```
Comment (paulmolluzzo) [inline pages/docs.js]: `borderRadius`

File: pages/docs.js [hunk 5]
```diff
@@ -31,7 +31,29 @@ const MarkdownWrapper = glamorous.div({
...
+    '& code': {
+      padding: '.3rem',
+      'border-radius': '.3rem'
+    }
+  },
+  '& blockquote': {
+    margin: '1rem 0',
+    padding: '0 1rem',
+    color: '#666',
+    'border-left': '6px solid #ddd'
```
Comment (paulmolluzzo) [inline pages/docs.js]: `borderLeft`

File: components/footer.js [hunk 6]
```diff
@@ -5,10 +5,11 @@ import * as colors from '../styles/colors'
 
 export default () => {
   return (
-    <glamorous.Footer position="fixed" bottom={0} right={0} left={0}>
+    <glamorous.Footer bottom={0} right={0} left={0}>
```
Comment (paulmolluzzo) [inline components/footer.js]: Maybe let's refactor this to match the same factory syntax in the `docs.js` file?

File: pages/docs.js [hunk 7]
```diff
@@ -31,7 +31,29 @@ const MarkdownWrapper = glamorous.div({
...
+  },
+
+  '& li': {
+    'line-height': '2rem',
+    "& code": {
+       padding: '.3rem',
+      'border-radius': '.3rem'
+    }
+  },
```
Comment (paulmolluzzo) [inline pages/docs.js]: Trailing comma and line

File: components/footer.js (hunk 6 above)
Comment (liadbiz) [inline components/footer.js]: what do you mean?

Comment (paulmolluzzo) [inline components/footer.js]: Like:

```js
import React from 'react'
import glamorous from 'glamorous'
import {Anchor} from '../components/styled-links'
import * as colors from '../styles/colors'

const Footer = glamorous.footer({
  position: 'fixed',
  bottom: 0,
  right: 0,
  left: 0,
  marginLeft: 'auto',
  marginRight: 'auto',
  width: '100%',
  maxWidth: 600,
  paddingTop: 10,
  paddingBottom: 10,
  fontSize: '0.6em',
  textAlign: 'center',
  borderTop: `1px solid ${colors.faded}`
})

export default () => {
  return (
    <Footer>
      Made with 💙 by <Anchor external href="https://github.com/paypal">PayPal</Anchor>.
    </Footer>
  )
}
```

IMHO, something like this is easier to read and maintain.

Comment (liadbiz) [inline components/footer.js]: yeah, I agree.  I will do it soon.

**Comment by paulmolluzzo** [comment]
@liadbiz This is great, and I'd love to merge as is! We just merged some other items into `master` that will not conflict, but will give us CI validation and also allow you to add yourself as a contributor. If you'd like to merge in the `master` branch and then add yourself as a contributor, that would be great. You can see how to add yourself here: https://github.com/kentcdodds/glamorous-website/blob/master/CONTRIBUTING.md#add-yourself-as-a-contributor

LMK if you get stuck or there's a big conflict when merging.

**Comment by liadbiz** [comment]
@paulmolluzzo 
I merged master from this repo to my fork, and when I run `npm start validate` there exists one  warings:
image

I noticed the lint rules `no-danger` are being set to 1, shoud we change it to 0?

**Comment by kentcdodds** [comment]
Just going to merge this as is. We can make any needed adjustments in another PR :+1:

**Comment by paulmolluzzo** [comment]
@liadbiz I use warnings there because we should keep an eye on` dangerouslySetInnerHTML`. Outside of a few controlled scenarios we really don't want it. We might even want to only disable those lines and keep the rule.

**Pull request merged**